swagger.yaml

```diff
@@ -48,6 +51,11 @@ paths:
           type: string
           required: true
           description: component version
+        - in: header
+          name: security-vendor
```

Comment:
If we are having a completely different endpoint (v2) then in that case do we need to pass this value?

Reply:
As we are planning to have the uuid generation at the server side, we should make sure that the component analysis too sends the uuid in its header. Initially it should send blank, and the same would be generated and sent back as the comp analysis response. The uuid can then be saved locally and sent in the subsequent calls.